packages/blade/src/components/Carousel/Carousel.stories.tsx

```diff
@@ -281,8 +281,8 @@ const CarouselTestimonialTemplate: ComponentStory<typeof CarouselComponent> = (p
   );
 };
 
-export const Default = CarouselTestimonialTemplate.bind({});
-Default.args = {
+export const DefaultCarousel = CarouselTestimonialTemplate.bind({});
```

Comment:
Changin this so that "Default" storyName isn't the same for multiple components. Example: Checkbox could have an export Default as well as Carousel so when we compose all the stories together, its hard to know which "Default" storyName is associated with which component. Having this separation helps in mapping storyName to component name in `BrandedComponentKitchenSink.tsx`